.eslintrc.js

```diff
@@ -71,7 +71,7 @@ module.exports = {
     // Enforces some naming conventions across the codebase.
     '@typescript-eslint/naming-convention': [
       2,
-      { leadingUnderscore: 'allow', selector: 'default', format: ['camelCase', 'UPPER_CASE', 'PascalCase'] },
+      { selector: 'default', leadingUnderscore: 'allow', format: ['camelCase', 'UPPER_CASE', 'PascalCase'] },
```

Comment:
🙃 Looks like a change that changes nothing

Reply:
yea, created while merging, didnt notice, i thought i took incoming changes

Reply:
That is exactly what i wanted to achieve - srly, thats caused of diff between what i added and what has been added into main earlier